.github/dockerfiles/ov_build/ubuntu_22_04_x64_nvidia/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM openvinogithubactions.azurecr.io/dockerhub/nvidia/cuda:11.8.0-runtime-ubuntu22.04
+
+USER root
+
+# APT configuration
+RUN echo 'Acquire::Retries "10";' > /etc/apt/apt.conf && \
+    echo 'APT::Get::Assume-Yes "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::Fix-Broken "true";' >> /etc/apt/apt.conf && \
+    echo 'APT::Get::no-install-recommends "true";' >> /etc/apt/apt.conf
+
+ENV DEBIAN_FRONTEND="noninteractive" \
+    TZ="Europe/London"
+
+RUN apt-get update && \
+    apt-get install software-properties-common && \
+    add-apt-repository --yes --no-update ppa:git-core/ppa && \
+    add-apt-repository --yes --no-update ppa:deadsnakes/ppa && \
+    apt-get update && \
+    apt-get install \
+        curl \
+        wget \
+        git \
+        ca-certificates \
+        gpg-agent \
+        tzdata \
+        # Python
+        python3.11-dev \
```

Comment:
why do not use default 3.12 version?

Reply:
All the test jobs were done with Python 3.11 in mind so I did not want to make this PR even bigger, I believe switching to a default system Python is a separate issue. I've done so in the U20 and introduced U24 Dockerfiles.